packs.json

```
@@ -8,6 +8,15 @@
         "position": 1,
         "size": 20
     },
+    {
+        "code": "ad",
+        "cycle_code": "ashes",
+        "date_release": "2019-03-15",
```

According to [this](http://nisei.net/article/HYPER01), Downfall is released 18th March, not 15th.
```suggestion
        "date_release": "2019-03-18",
```